### PR TITLE
DDCNL-3943 |play27 bootstrap upgrade

### PR DESCRIPTION
**Pull request by shikkhasingh**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -4,22 +4,19 @@ import play.core.PlayVersion
...
-  private val playVersion = "play-26"
+  private val playVersion = "play-27"
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% s"bootstrap-$playVersion" % "1.16.0",
-    "uk.gov.hmrc" %% "play-ui"                 % s"8.2.0-$playVersion",
-    "uk.gov.hmrc" %% "domain"                  % s"5.10.0-$playVersion",
-    "uk.gov.hmrc" %% "govuk-template"          % s"5.57.0-$playVersion"
+    "uk.gov.hmrc" %% s"bootstrap-frontend-$playVersion" % "4.3.0",
```
Comment (struong) [inline project/AppDependencies.scala:11]: Bump to 5.x please

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,22 +4,19 @@ import play.core.PlayVersion
...
+    "uk.gov.hmrc" %% "govuk-template"          % s"5.66.0-$playVersion"
   )
 
   val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"                       %% "hmrctest"              % s"3.9.0-$playVersion" ,
-    "org.scalatest"                     %% "scalatest"             % "3.0.8",
     "org.pegdown"                       %  "pegdown"               % "1.6.0",
-    "com.typesafe.play"                 %% "play-test"             % PlayVersion.current,
-    "org.scalatestplus.play"            %% "scalatestplus-play"    % "3.1.0",
+    "org.scalatestplus.play"            %% "scalatestplus-play"    % "3.1.3",
```
Comment (struong) [inline project/AppDependencies.scala:19]: Shouldn't this be 4.x for play 2.7?

File: conf/application.conf [hunk 3]
```diff
@@ -22,9 +22,9 @@ play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.frontend.FrontendModule"
 
-play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.MicroserviceFilters"
+play.http.filters = "uk.gov.hmrc.play.bootstrap.frontend.filters.FrontendFilters"
```
Comment (struong) [inline conf/application.conf:27]: Sorry missed this, this can now be deleted with the latest bootstrap play

**Review by struong — APPROVED** [review]

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/master/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Pull request merged**